src/kibana/components/index_patterns/index_patterns.js

```diff
@@ -24,11 +25,17 @@ define(function (require) {
       return (new IndexPattern(id)).init();
     };
     
-    self.refresh = function() {
+    self.refreshNetworkIndex = function() {
       return self.get(DEFAULT_INDEX_PATTERN).then(function(indexPattern){
          indexPattern.refreshFields();
       });
-   };
+    };
+
+    self.refreshEventsIndex = function() {
+      return self.get(EVENTS_INDEX_PATTERN).then(function(indexPattern){
+         indexPattern.refreshFields();
+      });
+    };
```

Comment:
Alex told me to create a second function for the events index rather than trying to reuse the first one. The next iteration of this code will be to programmatically determine which index is being used on the current page, and then just refreshing the current index instead of both